RecoCTPPS/PixelLocal/interface/RPixRoadFinder.h

```diff
@@ -0,0 +1,60 @@
+/*
+ *
+* This is a part of CTPPS offline software.
+* Author:
+*   Fabrizio Ferro (ferro@ge.infn.it)
+*   Enrico Robutti (robutti@ge.infn.it)
+*   Fabio Ravera   (fabio.ravera@cern.ch)
+*
+*/
+#ifndef RecoCTPPS_PixelLocal_RPixRoadFinder_H
+#define RecoCTPPS_PixelLocal_RPixRoadFinder_H
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/Common/interface/DetSet.h"
+//#include "DataFormats/Common/interface/DetSetVectorNew.h"
+//#include "FWCore/Framework/interface/EDProducer.h"
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

DetSetVectorNew is a preferred solution BTW, but for small sets it would not matter much.